gcp_variant_transforms/libs/bigquery_vcf_schema_converter.py

```diff
@@ -264,9 +264,13 @@ def _add_info_fields_from_alternate_bases(schema, infos):
   Notice that the validation of field mode is skipped for reserved fields since
   the mode (NULLABLE) of field in alternate bases is expected to be different
   from the mode (REPEATED) in reserved field definition.
+
+  Any `Record` field within alternate bases is considered as an annotation
+  field, and the annotation fields are skipped.
   """
   for field in schema.fields:
-    if field.name in _CONSTANT_ALTERNATE_BASES_FIELDS:
+    if (field.name in _CONSTANT_ALTERNATE_BASES_FIELDS or
```

Comment:
This seems unrelated to this PR; usually it's better to do it in another PR, but I leave it up to you as it's a small change anyway.